### DISCUSSION TITLE
command not found: npm

### DISCUSSION THREAD

**Discussion by CodeLifeITs** [saran-surya/email_auth_node#12] [Q&A] [answered]
am just trying to run the command "npm i && npm link" and i got the error "command not found: npm"

**Comment by saran-surya** [comment, marked as answer]
Hi @CodeLifeITs , It is because you have not installed npm in your local machine, Kindly install them from the following link

[Node JS for your system](https://nodejs.org/en/)

After that follow the same steps. I will also update the readme regarding the same. Thanks
